dashboard/src/components/PreservationActionCollapse.vue

```diff
@@ -4,54 +4,58 @@ import PackageReviewAlert from "@/components/PackageReviewAlert.vue";
 import StatusBadge from "@/components/StatusBadge.vue";
 import { useAuthStore } from "@/stores/auth";
 import Collapse from "bootstrap/js/dist/collapse";
-import { onMounted, watch } from "vue";
+import { onMounted, watch, ref, toRef } from "vue";
 import IconCircleChevronDown from "~icons/akar-icons/circle-chevron-down";
 import IconCircleChevronUp from "~icons/akar-icons/circle-chevron-up";
 
 const authStore = useAuthStore();
 
-const { action, index, toggleAll } = defineProps<{
+const props = defineProps<{
   action: api.EnduroPackagePreservationAction;
   index: number;
   toggleAll: boolean | null;
 }>();
 
+const action = toRef(props, "action");
+const index = toRef(props, "index");
+const toggleAll = toRef(props, "toggleAll");
```

Comment:
Would this work?

    const { action, index, toggleAll } = toRefs(props);

Reply:
😲 `toRefs`, it does.